docs/v1/2017_Modified_LAR_Data_Dictionary.csv

```diff
@@ -0,0 +1,37 @@
+Data Field Name,Data Field Type,Valid Values 

```

Comment:
Remove single quotes from NAs

Reply:
Please follow the change pattern listed in the comments on the 2018 Data Dictionary.csv so that the formats match.

Reply:
Sure! Single quotes for NA were reflected in the last commit.